### PR TITLE
Simplify bucket insertion for standard X-rays.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: xray/src/build_quadtree.rs [hunk 1]
```diff
@@ -142,7 +142,7 @@ pub fn run<T: Extension>(data_provider_factory: DataProviderFactory) {
             xray => ColoringStrategyKind::XRay,
             colored => ColoringStrategyKind::Colored(binning),
             colored_with_intensity => ColoringStrategyKind::ColoredWithIntensity(
-                value_t!(args, "min_intensity", f32).unwrap_or(1.),
+                value_t!(args, "min_intensity", f32).unwrap_or(0.),
                 value_t!(args, "max_intensity", f32).unwrap_or(1.),
```
Comment (krzesi-mir) [inline xray/src/build_quadtree.rs]: Wouldn't it make more sense to panic if provided arguments cannot be parsed?

File: xray/src/generation.rs [hunk 2]
```diff
@@ -180,16 +180,8 @@ impl ColoringStrategy for XRayColoringStrategy {
...
-                    e.get_mut().insert(d_loc.z);
-                }
-                Entry::Vacant(v) => {
-                    let mut set = FnvHashSet::default();
-                    set.insert(d_loc.z);
-                    v.insert(set);
-                }
-            }
+            let z_buckets = self.z_buckets.entry((d_loc.x, d_loc.y)).or_default();
+            z_buckets.insert(d_loc.z);
```
Comment (krzesi-mir) [inline xray/src/generation.rs]: nit: use one-liner?
```suggestion
            self.z_buckets.entry((d_loc.x, d_loc.y)).or_default().insert(d_loc.z);
```

File: xray/src/build_quadtree.rs (hunk 1 above)
Comment (krzesi-mir) [inline xray/src/build_quadtree.rs]: WOuldn't it make more sense to panic when the provided arguments cannot be parsed?

File: xray/src/generation.rs (hunk 2 above)
Comment (feuerste) [reply xray/src/generation.rs]: Tried this before, cargo fmt doesn't make it a one-liner, but a three- or four-liner :(

File: xray/src/build_quadtree.rs (hunk 1 above)
Comment (feuerste) [reply xray/src/build_quadtree.rs]: Problably, this was just some random stuff I observed... how about just removing the `_or()` part  and make it an `unwrap()`?

Comment (feuerste) [reply xray/src/build_quadtree.rs]: Hm, actually it makes most sense to provide default values above and change it here to `expect`s

Comment (krzesi-mir) [reply xray/src/build_quadtree.rs]: :+1: that makes sense.

File: xray/src/generation.rs (hunk 2 above)
Comment (krzesi-mir) [reply xray/src/generation.rs]: :+1:

File: xray/src/build_quadtree.rs (hunk 1 above)
Comment (feuerste) [reply xray/src/build_quadtree.rs]: Done, PTAL!

**Comment by feuerste** [comment]
💨 
Thanks for your review!

File: xray/src/build_quadtree.rs (hunk 1 above)
Comment (krzesi-mir) [reply xray/src/build_quadtree.rs]: lgtm

**Pull request merged**